Cargo.toml

```diff
@@ -9,12 +9,9 @@ members = [
 [workspace.dependencies]
 ijson = { git="https://github.com/RedisJSON/ijson", rev="e0119ac74f6c4ee918718ee122c3948b74ebeba8", default_features=false}
 serde_json = { version="1.0", features = ["unbounded_depth"]}
-# DO NOT CHANGE to avoid security issues. This exact version
-# specification is required for the second level dependencies
-# to use exactly this version too, so that the dependencies of
-# this project will not be able to compromise it.
-serde = { version = "=1.0.184", features = ["derive"] }
-serde_derive = "=1.0.184"
+
+serde = { version = "=1.0.203", features = ["derive"] }
```

Comment:
We can unfreeze serde if we port the cargo-deny support from the other projects. See: https://github.com/RedisGears/RedisGears/blob/master/deny.toml